java/Makefile.docker

```diff
@@ -20,4 +20,4 @@ dockerrun_pg ::
 
 dockerrun_ora ::
 	cp buildconf/test/rhn.conf.oracle-example buildconf/test/rhn.conf
-	docker run --privileged $(DOCKER_COMMON_OPTS) $(DOCKER_ORA_PORTS) $(DOCKER_VOLUMES) $(DOCKER_REGISTRY)/$(DOCKER_CONTAINER_BASE)-ora-4eclipse
+	docker run --privileged $(DOCKER_COMMON_OPTS) $(DOCKER_ORA_PORTS) $(DOCKER_VOLUMES) $(DOCKER_REGISTRY)/suma-head-ora-4eclipse
```

Comment:
Do we still need this container here if Uyuni won't support Oracle?

Reply:
The plan is to keep the "suma" container for a short period of time until we are 150% sure that nobody will request oracle comming back.

When this happens we need a new PR to remove all references to that container.